website_sale_google_shopping/controllers/website_sale_google_shopping.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+# License, author and contributors information in:
+# __openerp__.py file at the root folder of this module.
+from datetime import datetime, timedelta
+import openerp
+from functools import partial
+from openerp import http
+from openerp.http import request
+from openerp.addons.website_sale.controllers.main import get_pricelist
+
+
+DATE_FORMAT = '%Y-%m-%d %H:%M'
+FEED_MIMETYPE = 'application/xml; charset=utf-8'
+
+
+class GoogleShopping(http.Controller):
+    def get_effective_date(self, date_start, date_end):
+        if not date_start or not date_end:
+            return ''
+        ds = datetime.strptime(date_start, '%Y-%m-%d %H:%M:%S')
```

Comment:
Please use `fields.Datetime.from_string` for all datetime str<=>obj conversions.